src/shared/id128-print.c

```diff
@@ -34,8 +32,8 @@ int id128_pretty_print_sample(const char *name, sd_id128_t id) {
                on, SD_ID128_FORMAT_VAL(id), off,
                man_link,
                on, name);
-        for (i = 0; i < 16; i++)
-                printf("%02x%s", id.bytes[i], i != 15 ? "," : "");
+        for (unsigned i = 0; i < 16; i++)
```

Comment:
maybe switch to the more appropriate size_t here at the same time? after all its a simply memory offset, which generally is size_t for us

Reply:
It's iterating from 0 to 16… I changed it to `size_t`, but I don't think this matters.